temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
```

Comment:
The function currently only supports text and JSON files. Consider making it more generic to handle other file types as well. [medium]